cebra/solver/base.py

```diff
@@ -384,7 +384,7 @@ def num_total_features(self):
     def __post_init__(self):
         super().__post_init__()
         self._check_dimensions()
-        self.model = cebra.models.MultiobjectiveModel(
+        self.model = cebra.models.LegacyMultiobjectiveModel(
```

Comment:
isn't it better to rename the new variable and not cause a backwards-breaking change?

Reply:
I agree. How do we want to name the "new" multiobjective model, @MMathisLab ?

I think I'll still mark the old solver as deprecated.